### PR TITLE
Feature/audio preview

### PR DESCRIPTION
**Pull request by MrBrN197**
### Changes | 🦋
- ☑ Added Play/Pause Icon overlay on the modal image
- ☑ Added Play Preview audio for selected track functionality
- ☑ Create API placeholder data for debugging purposes.

localhost_8080_(iPad)

### CODE REVIEW

File: src/api/functions.js [hunk 1]
```diff
@@ -1,4 +1,28 @@
...
+//     },
+
+//     text: () => Promise.resolve(JSON.stringify([
+//       {
+//         item_id: '4wKGNANAC9OHjccAiihjGO',
+//         likes: 5,
+//       },
+//     ])),
+//   },
+// );
```
Comment (Whoistolu) [inline src/api/functions.js:26]: - [ ] It will be nice to remove all unused code in order to have a clean codebase. It's standard practice.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
### Hello @MrBrN197 and @Rayan84 , 

**This is a great project you have going on, well-done**. 
   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️

### To Highlight 
- [ ] It will be nice to remove all unused code in order to have a clean codebase.
 It's standard practice

_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/api/functions.js (hunk 1 above)
Comment (MrBrN197) [reply src/api/functions.js]: Thanks, 👍

**Review by Whoistolu — APPROVED** [review]
# STATUS: APPROVED 🟢
   
## Thank you for implementing all the project requirements, you've done a great job :trophy:  . There is nothing else to say other than... it's time to merge.

### Congratulations 🎉   

Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**